mediathread/settings_test.py

```diff
@@ -42,6 +42,8 @@
     'django_statsd.middleware.GraphiteMiddleware')
 MIDDLEWARE.remove(
     'impersonate.middleware.ImpersonateMiddleware')
+MIDDLEWARE.remove(
```

Comment:
Is the debug toolbar needed here?

Reply:
No - these settings are used for lettuce tests. I was just trying to simplify things.